app/client/rpc/server.go

```diff
@@ -0,0 +1,174 @@
+package rpc
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+//TODO(deblasis): still unused, need to gather feedback on RPC routes below before implementing everything, for now I copied some code from v0 over as a blueprint for style
```

Comment:
Let's split out the server and the cli into two PRs.

I'm personally always in favor of iterating on "non-functional" but small, clear and easy-to-approach code in main

Reply:
Agreed, I will address the current AIs and request review again when I have 2 separate PRs.